src/utils/plotConfigGetters.js

```diff
@@ -395,6 +397,27 @@ const getNonPivotedFunnelSpecificData = (plotConfig, data) => {
   return newData;
 };
 
+const getGroupedBarSpecificData = (plotConfig, data, isDataPivoted) => {
+  const xAxisDataKey = getXAxisDataKey(plotConfig);
+  const yAxisDataKey = getYAxisDataKey(plotConfig);
+  const categoryAxisDataKey = getCategoryAxisDataKey(plotConfig);
+  const pivotedData = pivotDataByDataKey(plotConfig, data, xAxisDataKey);
+  const formattedPivotedData = pivotedData.map((datum) => {
+    const builtData = datum.data.reduce((aggregate, currentDatum) => {
+      const categoryAxisValue = currentDatum[categoryAxisDataKey];
+      const yAxisValue = currentDatum[yAxisDataKey];
+      aggregate[categoryAxisValue] = yAxisValue;
+      return aggregate;
+    }, {});
+    return {
+      [xAxisDataKey]: datum.name,
```

Comment:
Note I'm passing this through to eventually be used by `FollowUpMenu`, because this piece of information is present in the same place for BarPlots, and I wanted the "shape" of info passed back by GroupedBar to match Bar if I need to use it in `FollowUpMenu`.